config/database.yml

```diff
@@ -21,6 +21,8 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password: io
```

Comment:
I vote to just leave username/password blank. Unfortunately, we need a `database.yml` or travis-ci won't run.